OcCo_TF/completion_models/pcn_cd.py

```diff
@@ -34,24 +34,23 @@ def create_decoder(self, features):
         with tf.variable_scope('decoder', reuse=tf.AUTO_REUSE):
             coarse = mlp(features, [1024, 1024, self.num_coarse * 3])
             coarse = tf.reshape(coarse, [-1, self.num_coarse, 3])
-
-        with tf.variable_scope('folding', reuse=tf.AUTO_REUSE):
-            grid = tf.meshgrid(tf.linspace(-0.05, 0.05, self.grid_size), tf.linspace(-0.05, 0.05, self.grid_size))
-            grid = tf.expand_dims(tf.reshape(tf.stack(grid, axis=2), [-1, 2]), 0)
-            grid_feat = tf.tile(grid, [features.shape[0], self.num_coarse, 1])
-
-            point_feat = tf.tile(tf.expand_dims(coarse, 2), [1, 1, self.grid_size ** 2, 1])
-            point_feat = tf.reshape(point_feat, [-1, self.num_fine, 3])
-
-            global_feat = tf.tile(tf.expand_dims(features, 1), [1, self.num_fine, 1])
-
-            feat = tf.concat([grid_feat, point_feat, global_feat], axis=2)
-
-            center = tf.tile(tf.expand_dims(coarse, 2), [1, 1, self.grid_size ** 2, 1])
-            center = tf.reshape(center, [-1, self.num_fine, 3])
-
-            fine = mlp_conv(feat, [512, 512, 3]) + center
-        return coarse, fine
+#        with tf.variable_scope('folding', reuse=tf.AUTO_REUSE):
```

Comment:
Is there any reason to keep this around at all? If we will not be using this section of code moving forward, better to delete it altogether than to leave a giant block of commented-out code in.